### PR TITLE
Skip fake link pop up banner for brazil client

### PR DESCRIPTION
**Pull request by thivya-deriv**
Card: https://app.clickup.com/t/20696747/BOT-1718
Desc: Add test to check on warning banner when country is set to Brazil.

### CODE REVIEW

**Review by bilalmajid-deriv — APPROVED** [review]

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]
can we also log the cookie value in the initial steps of the test? this can help us for future debugging if the pop up is appearing in non-Brazil countries.

File: cypress/support/commands/common.js [hunk 1]
```diff
@@ -608,3 +609,16 @@ Cypress.Commands.add('c_skipPasskeysV2', () => {
       }
     })
 })
+
+Cypress.Commands.add('c_fakeLinkPopUpCheck', () => {
+  cy.getCookie('website_status').then((cookie) => {
+    if (cookie && cookie.value === expectedCookieValue) {
```
Comment (abuzar-deriv) [inline cypress/support/commands/common.js:615]: Can you try with this? Just for simplicity :)
```suggestion
    if (cookie?.value === expectedCookieValue) {
```

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Comment by meeyun-deriv** [comment]
Please help to resolve conflicts

**Review by abuzar-deriv — APPROVED** [review]
LGTM

**Pull request merged**